### DISCUSSION TITLE
Justified Gallery每行图片数量

### DISCUSSION THREAD

**Discussion by leimao** [ppoffice/hexo-theme-icarus#1073] [Q&A]
不知道为啥，我的Justified Gallery每行默认显示5张图片，有参数可以设置改成[示例](https://ppoffice.github.io/hexo-theme-icarus/Plugins/Other/icarus%E7%94%A8%E6%88%B7%E6%8C%87%E5%8D%97-%E5%85%B6%E4%BB%96%E6%8F%92%E4%BB%B6/)里的每行3张吗？

![image](https://user-images.githubusercontent.com/17606112/167276299-9f4d1ce7-51c4-49ec-a48d-16711c0a9c0d.png)

**Comment by ppoffice** [comment]
@leimao 需要手动用JavaScript设置row height
https://miromannino.github.io/Justified-Gallery/options-and-events/
https://github.com/ppoffice/hexo-theme-icarus/blob/5.0.0-rc.1/source/js/main.js#L20

**Reply by leimao** [reply]
巧了，我正好在看这个。具体得修改[这一行](https://github.com/ppoffice/hexo-theme-icarus/blob/63cef9061d5721ef0c4ab63bc5019ee8e0df2f3a/source/js/main.js#L20)为
```
    if (typeof $.fn.justifiedGallery === 'function') {
        if ($('.justified-gallery > p > .gallery-item').length) {
            $('.justified-gallery > p > .gallery-item').unwrap();
        }
        $('.justified-gallery').justifiedGallery(
            {
                rowHeight: 200
            }
        );
    }
```
但是这是一个全局设置，希望以后能有interface这样可以局部设置。
